mitodl/ocw-hugo-themes
# ocw studio structure adjustments

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/160

#### What's this PR do?
In https://github.com/mitodl/ocw-to-hugo/pull/331 and https://github.com/mitodl/ocw-to-hugo/pull/332, we adjusted `ocw-to-hugo`'s output to more closely match what `ocw-studio` produces when exporting course site content.  This PR implements layout changes to adjust to this new unified structure.

#### How should this be manually tested?
 - Read the readme if you have never spun up a course site before using `ocw-hugo-themes` and make sure you are set up to download courses from S3
 - Make sure `OCW_TO_HUGO_PATH` and `OCW_TO_HUGO_OUTPUT_DIR` are blank in your `.env` and set `OCW_TEST_COURSE` to a `course_id` for any published course
 - Run `yarn install`
 - Spin up a course site with `npm run start:course`
 - Verify that the course site renders at http://localhost:3000 and you can click around to the different sections
